kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/release/ComparableReleaseId.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.server.services.impl.release;
+
+import org.apache.maven.artifact.DefaultArtifact;
+import org.apache.maven.artifact.handler.DefaultArtifactHandler;
+import org.kie.api.builder.ReleaseId;
+
+/**
+ * A release id that is comparable.
+ */
+public class ComparableReleaseId implements ReleaseId, Comparable<ReleaseId> {
+
+    private static final String TYPE = "jar";
+
+    private final DefaultArtifact artifact;
```

Comment:
Did you try to consider using org.kie.api.management.GAV instead of DefaultArtifact? That way you wouldn't have to bring another external dependency here.